packages/core/src/App/app.jsx

```diff
@@ -62,10 +62,6 @@ const AppWithoutTranslation = ({ root_store }) => {
 
         const loadExternalScripts = async () => {
             const delay = ms => new Promise(resolve => setTimeout(resolve, ms));
-
-            await delay(3000);
-            window.LiveChatWidget.init();
```

Comment:
Where do we now initialize LiveChat?

Reply:
in appcontent, with updated hook and after loading client-store, instead of just initi with settimeout